app/src/org/commcare/engine/cases/CaseUtils.java

```diff
@@ -64,11 +64,7 @@ public static void purgeCases() {
         }
 
         SQLiteDatabase db;
-        try {
-            db = CommCareApplication._().getUserDbHandle();
-        } catch (SessionUnavailableException e) {
-            throw new UserStorageClosedException(e.getMessage());
```

Comment:
Hm so this actually changes what exception is thrown; Are you sure that this `UserStorageClosedException` isn't caught and used to trigger specific behavior anywhere?